docs/features/kubernetes/configuration.md

```diff
@@ -261,6 +265,12 @@ Kubernetes plugin.
 
 Defaults to `false`.
 
+#### Custom `KubernetesClustersSupplier`
+
+If the configuration-based cluster locators do not work for your use-case,
+it is also possible to implement a
+[custom `KubernetesClustersSupplier`](installation#custom-cluster-discovery).
```

Comment:
I added this list to prevent a bit of confusion I had while reading the docs.  If it's out of scope for this MR I can remove this list again.